File: level32/homework/hw6.py
# 7) გააკეთეთ Sum Of Even Numbers. მიზანი: შეკრიბეთ ყველა ლუწი რიცხვი და შეინახეთ სიაში შემდეგ ეგ სია დაპრინტეთ 10 ჯერ

num = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]

sum_of_even_numbers = 0

for i in range(len(num)):
    if num[i] % 2 == 0: 
         sum_of_even_numbers += num[i]

print(sum_of_even_numbers)

File: level105/homework/hw6.js
// 6)დაწერე ფუნქცია, რომელიც იღებს სტუდენტების სახელების სიას და აბრუნებს(ითვლის) რამდენი სახელი იწყება ასო „გ“-ზე.

function func(students) {
  let sum = 0;
  for (let i = 0; i < students.length; i++) {
    if (students[i][0] == "გ") {
      sum++;
    }
  }
  console.log(sum);
}

func(["გიორგი", "ნიკა", "გვანცა", "ანა"]);